packages/wrangler/src/dev.tsx

```diff
@@ -273,6 +273,12 @@ export function devOptions(yargs: Argv): Argv<DevArgs> {
 				describe: "Use legacy environments",
 				hidden: true,
 			})
+			.option("log-level", {
+				// "none" will currently default to "error" for Wrangler Logger
```

Comment:
Can we get this on `wrangler pages dev` too please?

Reply:
Perhaps even a global flag.... ?

Reply:
`-v`, `-vv`, `-vvv` (verbose) like curl?

Reply:
Added to `pages dev`